tasks/arrays/arrays.js

```diff
@@ -35,6 +39,7 @@ function getJerryInfo(chars) {
  * @return {boolean}
  */
 function isAllHuman(chars) {
+    return  chars.map(char => char.species).every(species => species === "Human");
```

Comment:
it is superfluous and map() is not needed here:
return  chars.every(char => char.species === "Human");